README.md

```diff
@@ -1,3 +1,7 @@
+![Statements Covered!](./coverage/badge-statements.svg)
```

Comment:
You should also remove the lines in TODO for "add badges" and "publish to registries", and while you're at it, correct the end of the sentence just above that that says "for use in production use."